usr/share/rear/prep/GNU/Linux/400_guess_kernel.sh

```diff
@@ -34,6 +34,9 @@ for dummy in "once" ; do
     # Try /boot/vmlinuz-$KERNEL_VERSION:
     KERNEL_FILE="/boot/vmlinuz-$KERNEL_VERSION"
     test -s "$KERNEL_FILE" && continue
+    # ppc64el uses uncompressed kernel
+    KERNEL_FILE="/boot/vmlinuz-$KERNEL_VERSION"
```

Comment:
This is the same KERNEL_FILE assignment as before.
I guess in this case ...vmlinux... is meant to be used?

Reply:
I think it is a typo... and should be vmlinux instead

Reply:
Hi, sorry guys, just reading the feedback, and indeed my pull request is wrong : uncompressed linux kernel is usually named vmlinux.
I updated the commit in my branch.

F.